routes/error-tracker.js

```diff
@@ -38,225 +34,145 @@ const SEVERITY = {
 };
 
 /**
- * @param {string} stackTrace
- * @param {string} version
- * @return {string} Stacktrace with all the v0.js urls versioned
- * - 'at     error https://cdn.ampproject.org/v0.js:5:314' becomes
- *  'at     error https://cdn.ampproject.org/rtv/031496877433269//v0.js:5:314'
- */
-function versionStackTrace(stackTrace, version) {
-  return stackTrace.replace(versionRegex, function(match, group1) {
-    if (!group1) {
-      return '/rtv/' + version + match;
-    }
-    return match;
-  });
-}
-
-/**
- * @param {string} stackTrace
- * @return {boolean} True if its a non JS stack trace
- */
-function isNonJSStackTrace(stackTrace) {
-  return stackTrace.split('\n').some(function(line) {
-    return !!line && !/\.js:\d+:\d+/.test(line);
-  });
-}
-
-/**
- * @param {string} message
- * @param {string} stack
- * @return {boolean}
- */
-function ignoreMessageOrException(message, stack) {
-  return errorsToIgnore.some(function(msg) {
-    return message.includes(msg) || stack.includes(msg);
-  });
-}
-
-/**
- * Converts a stack trace to the standard Chrome stack trace format.
- * @param {string} stackTrace
- * @return {string} The converted stack trace.
- */
-function standardizeStackTrace(stackTrace) {
-  if (chromeStackTraceRegex.test(stackTrace)) {
-    // Discard garbage stack trace lines
-    return stackTrace.match(chromeStackTraceRegex).join('\n');
-  }
-  let validStackTraceLines = [];
-  let match;
-  while ((match = mozillaSafariStackTraceRegex.exec(stackTrace))) {
-    if (match[1]) {
-      validStackTraceLines.push(
-          ` at ${match[1]} (${match[2]}:` +
-          `${match[3]}:${match[4]})`);
-    } else {
-      validStackTraceLines.push(
-          ` at ${match[1]} ${match[2]}:` +
-          `${match[3]}:${match[4]}`);
-    }
-  }
-  return validStackTraceLines.join('\n');
-}
-
-/**
  * Extracts relevant information from request, handles edge cases and prepares
  * entry object to be logged and sends it to unminification.
  * @param {Request} req
  * @param {Response} res
  * @return {?Promise} May return a promise that rejects on logging error
  */
-function getHandler(req, res) {
+function handler(req, res) {
   const params = req.query;
-  let stack = params.s || '';
+  const referrer = req.get('Referrer');
+  const version = params.v;
+  const message = params.m;
 
-  if (!params.r || !params.v || !params.m) {
+  if (!referrer || !version || !message) {
     res.sendStatus(statusCodes.BAD_REQUEST);
     return null;
   }
-  if (params.v.includes('$internalRuntimeVersion$')) {
-    res.sendStatus(statusCodes.NO_CONTENT);
+  if (version.includes('$internalRuntimeVersion$')) {
+    res.sendStatus(statusCodes.OK);
     return null;
   }
-  if (ignoreMessageOrException(params.m, stack)) {
-    res.set('Content-Type', 'text/plain; charset=utf-8');
-    res.status(statusCodes.BAD_REQUEST);
-    res.send('IGNORE');
+
+  const stack = standardizeStackTrace(params.s || '');
```

Comment:
Sure we want to standardize this stack trace so early? Will probably be throwaway work if we do it before throttling.